inst/rmarkdown/templates/measles_outbreak/skeleton/skeleton.Rmd

```diff
@@ -531,7 +532,7 @@ linelist_cleaned <- linelist_cleaned %>%
            )
 
 # Create a factor variable based on rules from other simple character variables
-# create a case definition variable 
+# If you have access to lab results, you can create a case definition variable 
```

Comment:
add a section creating a fake lab list and merging it based on ID. e.g. 

# THIS CAN BE DELETED IF YOUR DATA HAS THIS ALREADY A DIFFERENT DATASET 
# its just to be able to demonstrate posibilities

# generate artificial lab tests, symptoms and contact vars
lab_results <- linelist_cleaned %>% 
                select(case_id) %>% 
                mutate(test_result = sample(c("Positive", "Negative"),
                                            nrow(linelist_cleaned), 
                                            replace = TRUE), 
                       symptoms = sample(c("Yes", "No"),
                                            nrow(linelist_cleaned), 
                                            replace = TRUE),
                       contact = sample(c("Yes", "No"),
                                            nrow(linelist_cleaned), 
                                            replace = TRUE)
                )

# merging linelist with lab dataset 
linelist_cleaned <- left_join(linelist_cleaned, lab_results, 
                              by = "case_id")

Reply:
done